http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -139,17 +147,24 @@ class GKEInterpreter[F[_]](
 
       networkConfig = new com.google.api.services.container.model.NetworkConfig()
         .setEnableIntraNodeVisibility(params.enableIntraNodeVisibility)
+
+      networkPolicy =
+        if (params.autopilot) null else new com.google.api.services.container.model.NetworkPolicy().setEnabled(true)
```

Comment:
Why is the network policy null in case of autopilot?

Reply:
this is not supported in autopilot mode